ui/components/ui/icon/preloader/preloader-icon.component.js

```diff
@@ -19,7 +19,7 @@ const Preloader = ({ className, size }) => (
     />
     <mask
       id="mask0"
-      mask-type="alpha"
+      style={{ maskType: 'alpha' }}
```

Comment:
?

Reply:
linter was complaining about this. Unsure what changed about the linter or packages to cause this though

Reply:
there are other places in code where `style={{ maskType: 'alpha' }}` is used instead of `mask-type="alpha"`, so i figured this change was fine

Reply:
No reason to make this change as part of this PR though. This is an easy opportunity to reduce the scope of the PR.

Reply:
I also not seeing any lint errors for this 🤔

Reply:
Removed in [808cf97](https://github.com/MetaMask/metamask-extension/pull/27782/commits/808cf9717c2d5d481a856ab4d29a46ca9ec2e1e1), we'll see what happens.